Analytics/Analytics.csproj

```diff
@@ -29,6 +29,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json, Version=7.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
`Private=True` will just make the assembly be copied locally. I think the underlying issue is that a project with multiple versions of JSON.NET will fail because Segment is not explicitly specifying what version it expects. This change corrects that by using the fully-qualified reference.